README.md

```diff
@@ -106,14 +106,18 @@ Another source of light are electric lamps, such as incandescent light bulbs.
 They generate light by thermal emission from a very hot tungsten filament in a glass envelope.
 In physics, the spectral properties of thermal emission is described by Planck's law.
 For incandescent light bulbs, the CIE recommends using the A-illuminant, in this library available as [`StdIlluminant::A`](crate::std_illuminants::StdIlluminant::A).
+To use this illuminant you need to enable the `cie-illuminants` feature on this crate.
 
 For example, to get the A illuminant's chromaticity:
 ```rust
+    # #[cfg(feature="cie-illuminants")]
+    # {
```

Comment:
A line starting with a `#` in a code example is used when `cargo test` and `cargo doc` compiles the code to validate it is correct, BUT it does not show up in the generated documentation. This allows tweaking examples like this to make them build, but without including what is just noise in the documentation. Makes the docs easier to read but still compile.